### DISCUSSION TITLE
Cancel "onColResize" event

### DISCUSSION THREAD

**Discussion by jaelbosonit** [primefaces/community#1086] [PrimeNG] [answered]
Hi!
So basically what I need to do is to stop the resize event when you try to reduce it to a certain width so it's not so small.
I tried the stopPropagation and preventDefault functions when the resize event fires and don´t pass the minimum width condition but nothing stops the event!

`private stopPropagationFunc = (event: Event) => {
    event.stopPropagation();
    event.preventDefault();
  };`

`let cosa: any =
        Array.from(
          this.hostElem.nativeElement.querySelectorAll('.p-column-resizer')
        ) || [];`
`cosa.forEach((elem: Element) => {
        elem.addEventListener('mousedown', this.stopPropagationFunc);
      });`

Basically to do this:
`onColResize(event: any) {
    if (event.nativeElement.offsetWidth < 120) {
      // stop event
    }
  `

**Comment by cetincakiroglu** [comment, marked as answer]
Hi,

Could you please try it by assigning `min-width` value to the cell element. See the example https://stackblitz.com/edit/bprbkw?file=src%2Fapp%2Fdemo%2Ftable-column-resize-fit-mode-demo.html
